static/src/js/components/CustomSelectWidget/CustomSelectWidget.jsx

```diff
@@ -113,13 +116,17 @@ const CustomSelectWidget = ({
     if (enumOptions) {
       setSelectOptions(buildOptions(enumOptions))
     }
+
+    if (oneOfEnums) {
```

Comment:
isn't this always going to be true since line 100 has `|| {}`

Reply:
Yes correct, that's why I have the oneOfEnum condition at the bottom of the useEffect().

Reply:
Well if it is always true, then the if statement is not necessary :) unless I'm missing something.